.gitignore

```diff
@@ -1,2 +1,8 @@
 node_modules
-package-lock.json
```

Comment:
When starting with a fresh `git clone` I was getting this error:
```
npm ERR! Unable to authenticate, need: BASIC realm="Sonatype Nexus Repository Manager"
```
<img width="753" alt="image" src="https://github.com/TrustlyInc/trustly-react-native-example/assets/13320316/13580a96-2d2b-4bb6-945e-13a69d13e4e9">

After deleting `package-lock.json` and `yarn.lock` files everything worked fine. I need to confirm this behavior on other systems, but something to consider.